src/TheThingsNetwork.cpp

```diff
@@ -324,9 +324,20 @@ int TheThingsNetwork::poll(int port, bool confirm) {
   return sendBytes(payload, 1, port, confirm);
 }
 
-void TheThingsNetwork::showAirTime() {
-  debugPrint(F("Air-time: "));
-  debugPrintLn(this->timer / 1000);
+void TheThingsNetwork::airTimeValue() {
+  int payloadSize = 21;
+  int sf = TTN_DEFAULT_SF;
+  int ps = 8;
+  int band = 125;
+
+  float Tsym = pow(2, sf) / band;
+  float Tpreamble = (ps + 4.25) * Tsym;
+  unsigned int payLoadSymbNb = 8 + (max(ceil((8 * payloadSize - 4 * sf + 28 + 16 - 20 * (0))/ (4 * (sf - 2 * 1))) * (4), 0));
+  float Tpayload = payLoadSymbNb * Tsym;
+  float Tpacket = Tpreamble + Tpayload;
+  this->AirTimePerPackets = Tpacket;
+  this->messDay = (floor(30000 / Tpacket));
+  this->messHour = (30000 / 24 / Tpacket);
```

Comment:
@johanstokking messDay and messHour are just tests, I forgot to delete them... but the rest is there to calculate the airtime:  ``` this->AirTimePerPackets = Tpacket; ```